src/main/java/com/appium/ios/IOSDeviceConfiguration.java

```diff
@@ -154,23 +152,8 @@ public boolean checkiOSDevice() throws Exception {
 
     public String startIOSWebKit() throws IOException, InterruptedException {
         setIOSWebKitProxyPorts();
-        String serverPath = prop.getProperty("APPIUM_JS_PATH");
-        File file = new File(serverPath);
-        File currentPath = new File(file.getParent());
-        System.out.println(currentPath);
-        file = new File(currentPath + "/.." + "/..");
-        String ios_web_lit_proxy_runner =
-                file.getCanonicalPath() + "/bin/ios-webkit-debug-proxy-launcher.js";
-        int port = 0;
-        try {
-            port = ap.getPort();
-        } catch (Exception e) {
-            e.printStackTrace();
-        }
-        String webkitRunner =
-                ios_web_lit_proxy_runner + " -c " + DeviceManager.getDeviceUDID()
-                        + ":" + port + " -d";
-        System.out.println(webkitRunner);
+        String webkitRunner = "ios_webkit_debug_proxy -c " + DeviceManager.getDeviceUDID() + ":"
```

Comment:
@shridharkalagi Can we somehow throw error if user doesnt install iwdp or incase if iwdp doesn't start properly?